cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/TableAdmin.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.spi.data;
+
+import co.cask.cdap.spi.data.table.TableSpecification;
+
+/**
+ * Defines admin operations on a {@link StructuredTable}
+ */
+public interface TableAdmin {
```

Comment:
Better call this `StructuredTableAdmin` to be consistent with `StructuredTable`?

Reply:
Done